packages/autoupdate/autoupdate_server.js

```diff
@@ -52,7 +52,7 @@ Autoupdate.autoupdateVersionRefreshable = null;
 Autoupdate.autoupdateVersionCordova = null;
 Autoupdate.appId = __meteor_runtime_config__.appId = process.env.APP_ID;
 
-var syncQueue = new Meteor._SynchronousQueue();
+var syncQueue = new Meteor._AsynchronousQueue();
```

Comment:
Maybe we should name it asyncQueue or just queue

Reply:
The queue only allows a single task to run at a time, which is why it was called a synchronous queue. The name wasn't referring to if the tasks themselves were sync or async; it always supported async tasks.

Reply:
ohhhhhh got it. Thanks for the clarification